backend/migrations/2023-06-20-110200_plants_relations/up.sql

```diff
@@ -0,0 +1,12 @@
+-- Your SQL goes here
+CREATE TYPE RELATIONS_TYPE AS ENUM ('companion', 'neutral', 'antagonist');
```

Comment:
We make enums lower-case and singular, see doc/guidelines/database.md and #311

```suggestion
CREATE TYPE relation AS ENUM ('companion', 'neutral', 'antagonist');
```

Reply:
I just copied that format from layers_relation_table migration. I would now leave it at `relation_type` otherwise it might get confusing with possible entities and DTOs.

Reply:
`relation_type` is fine.